### DISCUSSION TITLE
Timestamp with Snapshot

### DISCUSSION THREAD

**Discussion by cylstudio** [tsightler/ring-mqtt#785] [Feature Requests]
Hi, not sure if it's already implemented, just wondering if it's possible to have some sort of timestamp on top of the Snapshot image triggered by the Motion please?

**Comment by tsightler** [comment]
There is already an attribute that includes the timestamp of the snapshot.  It should be possible to use native HA functionality to add an overlay to the image using this attribute to calculate the timestamp, but I don't plan to add it to the image directly.
